src/rubicon/objc/api.py

```diff
@@ -246,28 +246,34 @@ def __repr__(self):
         return f"{type(self).__qualname__}({self.name_start!r})"
 
     def __call__(self, receiver, first_arg=_sentinel, **kwargs):
-        if first_arg is ObjCPartialMethod._sentinel:
+        # Ignore parts of argument names after "__".
+        order = [argname.split("__")[0] for argname in kwargs]
+        args = [arg for arg in kwargs.values()]
+
+        if first_arg is self._sentinel:
```

Comment:
```suggestion
        if first_arg is ObjCPartialMethod._sentinel:
```
Although it's technically equivalent, referencing the sentinel on the class highlights that it is a class-level object (I think it will be *marginally* faster, as well)